__tests__/materials/material.mutation.test.js

```diff
@@ -0,0 +1,440 @@
+                  value
+                }
+                description {
+                  lang
+                  value
+                }
+                purpose
+
+                colors {
+                  code
+                  name {
+                    lang
+                    value
+                  }
+                  simpleName {
+                    lang
+                    value
+                  }
+                  available
+                  images {
+                    large
+                    medium
+                    small
+                    thumbnail
+                  }
+                }
+                additionalPrice {
+                  currency
+                  value
+                }
+                available
+              }
+              ... on Error {
+                message
+                statusCode
+              }
+            }
+          }
+        `,
+      })
+      .catch(e => e);
+    const newMaterial = res.data.addMaterial;
+    materialId = newMaterial._id;
+
+    expect(newMaterial).toBeDefined();
+    expect(newMaterial).toHaveProperty('name', [
+      { __typename: 'Language', lang: 'uk', value: 'Тест mutation' },
+      { __typename: 'Language', lang: 'en', value: 'Test mutation' },
+    ]);
+    expect(newMaterial.name).toBeInstanceOf(Array);
+
+    expect(newMaterial).toHaveProperty('description', [
+      {
+        __typename: 'Language',
+        lang: 'uk',
+        value: 'Опис test',
+      },
+      {
+        __typename: 'Language',
+        lang: 'en',
+        value: 'Description test',
+      },
+    ]);
+    expect(newMaterial.description).toBeInstanceOf(Array);
+
+    expect(newMaterial).toHaveProperty('purpose', 'test');
+
+    expect(newMaterial).toHaveProperty('available', true);
+
+    expect(newMaterial).toHaveProperty('additionalPrice', [
+      { __typename: 'CurrencySet', currency: 'UAH', value: 0 },
+      { __typename: 'CurrencySet', currency: 'USD', value: 0 },
+    ]);
+    expect(newMaterial.additionalPrice).toBeInstanceOf(Array);
+
+    expect(newMaterial).toHaveProperty('colors', [
+      {
+        __typename: 'Color',
+        code: 777,
+        name: [
+          { __typename: 'Language', lang: 'uk', value: 'Тестовий колір' },
+          { __typename: 'Language', lang: 'en', value: 'Test color' },
+        ],
+        images: {
+          __typename: 'ImageSet',
+          large: 'large_test',
+          medium: 'medium_test',
+          small: 'small_test',
+          thumbnail: 'thumbnail_test',
+        },
+        available: true,
+        simpleName: [
+          {
+            __typename: 'Language',
+            lang: 'uk',
+            value: 'проста назва кольору',
+          },
+          { __typename: 'Language', lang: 'en', value: 'simple color name' },
+        ],
+      },
+    ]);
+    expect(newMaterial.colors).toBeInstanceOf(Array);
+  });
+  test('#2 adding existing material should return error', async () => {
```

Comment:
Please, write here something like `adding material with the existing name`, it ii much better reflects validation

Reply:
fixed